### PR TITLE
feat: change to generate finschia-std library using tag in env file

### PR DESCRIPTION
**Pull request by kokeshiM0chi**
# Overview
The submodule version currently uses a fixed value.
In this repository, we are going to update CI to keep automatically the tags of the env file the latest ones.  This change is in preparation for CI processing.

# Detail

This PR resolves sub issue "Manage tags in env file and run proto-build using that tag" of the issue #7 

Added a new file:
- finschia-wasm/env
Tags are written in the env file.

### CODE REVIEW

File: packages/proto-build/src/main.rs [hunk 1]
```diff
@@ -5,30 +5,12 @@
 
 use std::{env, path::PathBuf};
 
+use dotenvy;
```
Comment (loloicci) [inline packages/proto-build/src/main.rs:8]: It looks `dotenvy` cannot be imported. Declare to use it in Cargo.toml.

Comment (kokeshiM0chi) [reply packages/proto-build/src/main.rs:8]: I forgot to commit. thank you.

**Comment by da1suk8** [comment]
> In this repository, CI automatically updates tags to the latest ones. This is a PR for that purpose.

Where is the CI that automatically updates the env file with the tag version located?

**Comment by kokeshiM0chi** [comment]
@da1suk8 
I will separate the PR that rewrites the env file with CI. Thank you for confirming.

**Review by loloicci — APPROVED** [review]

**Review by da1suk8 — APPROVED** [review]

**Pull request merged**